test_tools/scripts/record_video_from_camera.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+
+import sys
+import rospy
+import cv2
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import Image
+
+
+class ImageConverter:
+    def __init__(self, topic_name, file_name, fps=20):
+        self.fps = fps
+        self.file_name = file_name
+        self.bridge = CvBridge()
+        self.image_sub = rospy.Subscriber(topic_name, Image, self.callback)
+        self.video_writer = None
+
+    def callback(self, data):
+        try:
+            cv_image = self.bridge.imgmsg_to_cv2(data, "bgr8")
+        except CvBridgeError as e:
+            rospy.logerr(e)
+        else:
+            if self.video_writer is None:
+                rospy.loginfo(f"Got image on topic, creating video writer with {self.fps} fps")
+                rows, cols, _ = cv_image.shape
+                fourcc = cv2.VideoWriter_fourcc(*"MJPG")
+                self.video_writer = cv2.VideoWriter(self.file_name, fourcc, self.fps, (cols, rows))
+
+            self.video_writer.write(cv_image)
+
+    def clean_shutdown(self):
+        if self.video_writer is not None:
+            self.video_writer.release()
+            rospy.loginfo(f"Saving video file {self.file_name}")
+        else:
+            rospy.loginfo(f"No video saved, no image received on topic")
+
+    def __del__(self):
+        self.clean_shutdown()
+
+
+def main():
+    image_topic = "/hero/head_rgbd_sensor/rgb/image_raw"
+    video_name = "video.avi"
+    if len(sys.argv) > 1:
+        image_topic = sys.argv[1]
+        if len(sys.argv) > 2:
+            video_name = sys.argv[2]
```

Comment:
```suggestion
    myargv = rospy.myargv()
    if len(myargv) > 1:
        image_topic = myargv[1]
        if len(myargv) > 2:
            video_name = myargv[2]
```

@PetervDooren luckily I didn't became lazy :p